public/jwpltx.js

```diff
@@ -90,13 +90,16 @@ window.jwpltx = window.jwpltx || {};
   }
 
   // Process a player ready event
-  o.ready = function (aid, bun, fed, id, t, oaid) {
+  o.ready = function (aid, bun, fed, id, t, oaid, oiid) {
     uri = JSON.parse(JSON.stringify(URI));
     uri.aid = aid;
     uri.bun = bun;
     uri.fed = fed;
     uri.id = id;
     uri.t = t;
+    uri.oiid = oiid;
+    // eslint-disable-next-line no-undef
+    uri.av = APP_VERSION;
```

Comment:
I would prefer to pass it as an argument (the value we get from `import.meta.env.APP_VERSION`);